desktop/core/src/desktop/conf.py

```diff
@@ -205,6 +205,18 @@ def is_jwt_authentication_enabled():
   type=str,
   default="SAMEORIGIN")
 
+HUE_IMAGE_VERSION = Config(
+  key="hue_image_version",
+  help=_("Image version of Hue"),
+  type=str,
+  default="")
+
+HUE_HOST_NAME = Config(
+  key="selected_hue_vw",
```

Comment:
Can you also change the key to not include `vw`?

Reply:
renamed it to key="hue_host".